pmoves/services/agentgym-rl-coordinator/app.py

```
@@ -0,0 +1,68 @@
+logger = logging.getLogger("agentgym-coordinator")
+
+# Configuration
+NATS_URL = os.getenv("NATS_URL", "nats://nats:4222")
+HF_TOKEN = os.getenv("HF_TOKEN")
+PORT = int(os.getenv("PORT", 8114))
+
+# Global NATS Client
+nc = None
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)